### PR TITLE
fix: Visibility and zooming when canvas offset is not zero

### PR DESCRIPTION
**Pull request by Jnforja**
This PR fixes 2 bugs:
1. bug described on issue #2528
2. Zooming on cursor not working when offset was not 0

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -2586,8 +2588,8 @@ class App extends React.Component<ExcalidrawProps, AppState> {
       );
 
       /* If arrow is pre-arrowheads, it will have undefined for both start and end arrowheads.
-			 If so, we want it to be null for start and "arrow" for end. If the linear item is not 
-			 an arrow, we want it to be null for both. Otherwise, we want it to use the 
+			 If so, we want it to be null for start and "arrow" for end. If the linear item is not
+			 an arrow, we want it to be null for both. Otherwise, we want it to use the
 			 values from appState. */
```
Comment (lipis) [inline src/components/App.tsx:2593]: How come we have tabs here :D

Comment (Jnforja) [reply src/components/App.tsx:2593]: That's strange... I'm not sure, but I did have some issues with Prettier while making this fix.

Comment (Jnforja) [reply src/components/App.tsx:2593]: Hmm... I'm a bit confused. Should I revert this change?

Comment (lipis) [reply src/components/App.tsx:2593]: Convert it to spaces

Comment (lipis) [reply src/components/App.tsx:2593]: Prettier is not changing much in the comments as far as I know.

Comment (dwelle) [reply src/components/App.tsx:2593]: there's an eslint rule preventing mixed tabs/spaces IIRC, if anyone wants to spend time on it :)

Comment (lipis) [reply src/components/App.tsx:2593]: Will take a look.. but let's remove them manually here for now.

Comment (lipis) [reply src/components/App.tsx:2593]: > there's an eslint rule preventing mixed tabs/spaces IIRC, if anyone wants to spend time on it :)

https://eslint.org/docs/rules/no-mixed-spaces-and-tabs doesn't seem to affect comments as well.

Comment (Jnforja) [reply src/components/App.tsx]: It's converted it so spaces :)

**Review by lipis — APPROVED** [review]

**Comment by dwelle** [comment]
Works well. Thanks @Jnforja!

**Pull request merged**